heft-plugins/heft-esbuild-plugin/schemas/esbuild.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Esbuild Build Configuration",
+  "description": "Defines optional options for Esbuild build.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "extends": {
+      "description": "Optionally specifies another JSON config file that this file extends from.  This provides a way for standard settings to be shared across multiple projects.",
+      "type": "string"
+    },
+    "builds": {
+      "type": "array",
+      "description": "The array of builds that Esbuild should run",
+      "items": {
+        "type": "object",
+        "properties": {
+          "entryPoints": {
+            "type": "array",
+            "description": "The array of entrypoints that Esbuild should compile",
+            "items": {
+              "type": "string"
+            }
+          },
+          "outdir": {
```

Comment:
```suggestion
          "outputFolder": {
```

Reply:
I'd prefer to keep these parameters matching the esbuild parameters so the docs are easily understood I think? e.g. https://esbuild.github.io/api/#outdir